modules/tinymce/src/core/test/ts/browser/selection/GetSelectionContentTest.ts

```diff
@@ -123,6 +136,15 @@ describe('browser.tinymce.selection.GetSelectionContentTest', () => {
     assertGetContent('Should be some content', editor, 'This Has Spaces', { format: 'text' });
   });
 
+  it('TINY-7981: inputs should not be deselected', () => {
+    const editor = hook.editor();
+    // eslint-disable-next-line max-len
+    editor.setContent('<p><label><input name="group-name" type="radio" value="1">Option 1</label><label><input checked="checked" name="group-name" type="radio" value="2">Option 2</label><label><input name="group-name" type="radio" value="3">Option 3</label></p>');
```

Comment:
Are the labels required here could the code fit on one line if they where removed. The bug is about input elements with the same name and checked state maybe value could be removed as well.